truss/util/path.py

```diff
@@ -86,19 +85,6 @@ def build_truss_target_directory(stub: str) -> Path:
     return target_directory_path
 
 
-def calc_shadow_truss_dirname(truss_path: Path) -> str:
-    resolved_path_str = str(truss_path.resolve())
-    return str_hash_str(resolved_path_str)
-
-
-def build_truss_shadow_target_directory(stub: str, truss_path: Path) -> Path:
```

Comment:
Removed because `git grep build_truss_shadow_target_directory` returns no results

Reply:
thanks for this cleanup